### PR TITLE
Fix for broken Android Studio Builds. Migrated to Androix X. Updated …

### PR DESCRIPTION
**Pull request by mitchross**
Fix for broken Android Studio Builds. Migrated to Androix X. Updated Gradle. Change Target to 29.

### CODE REVIEW

File: example/.flutter-plugins-dependencies [hunk 1]
```diff
@@ -0,0 +1 @@
+{"_info":"// This is a generated file; do not edit or check into version control.","dependencyGraph":[{"name":"flutter_vlc_player","dependencies":[]}]}
```
Comment (solid-software) [inline example/.flutter-plugins-dependencies]: It looks like this file shouldn't be a part of version control

Comment (mitchross) [reply example/.flutter-plugins-dependencies]: Good call, I fill fix. First time doing Flutter. I should set up a flutter gitignore.

**Comment by mitchross** [comment]
@solid-software resolved.

**Comment by solid-software** [comment]
Thank you @mitchross

**Pull request merged**